corehq/apps/app_manager/templates/app_manager/partials/modules/case_list_multi_select.html

```diff
@@ -11,13 +19,29 @@ <h4 class="panel-title panel-title-nolink">
     </div>
     <div class="panel-body">
       <div class="row">
-        <div class="col-sm-8">
```

Comment:
Why is this removed?

Reply:
The spacing seemed off, especially with the new input and I couldn't figure out why it was there to start-- is that standard?

Reply:
Yeah, it's standard to have a `row` div then contain a set of `col-X-Y` divs. Bootstrap has a max of 12 cols, but we often use 8 as the max so that the UI only takes up two-thirds of the screen.

I'm curious if it would work with putting each checkbox in its own row, probably with a col-sm-8 inside of it (either with the col-sm8 as a layer between the row and the checkbox, or adding the col-sm-8 to the checkbox div), but it's not worth spending time on if you've already futzed with it a bunch.